### PR TITLE
[parallelstl] update to 20200330

### PR DESCRIPTION
**Pull request by isanych**
version on master complaining about incorrect hash, so fixed by updating to latest version

### CODE REVIEW

**Review by LilyWangL — APPROVED** [review]

File: ports/parallelstl/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    REPO intel/parallelstl #20191218
-    REF  37761e15f62c03b7eb179db64ef930a69f33dfac
-    SHA512 dc8c875f692043b6f10ce7faf034e07536978537ea2a3d545be499cfa047f0c362c5109eb744a41dc0c92bd0fc56d98505183444bd376aeb257c9204480e6668
+    REPO intel/parallelstl #20200330
```
Comment (strega-nil) [inline ports/parallelstl/portfile.cmake:3]: This comment should probably be on the `REF`

Comment (strega-nil) [reply ports/parallelstl/portfile.cmake:3]:
```suggestion
    REPO intel/parallelstl
```

File: ports/parallelstl/portfile.cmake [hunk 2]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    REPO intel/parallelstl #20191218
-    REF  37761e15f62c03b7eb179db64ef930a69f33dfac
-    SHA512 dc8c875f692043b6f10ce7faf034e07536978537ea2a3d545be499cfa047f0c362c5109eb744a41dc0c92bd0fc56d98505183444bd376aeb257c9204480e6668
+    REPO intel/parallelstl #20200330
+    REF  0241743d73bb405b16d9c4a24b693e4533dc34a7
```
Comment (strega-nil) [inline ports/parallelstl/portfile.cmake:4]:
```suggestion
    REF  0241743d73bb405b16d9c4a24b693e4533dc34a7 # 20200330

```

**Comment by strega-nil** [comment]
Thanks @isanych!

**Pull request merged**